sdlf-utils/ingestion-examples/preprocessing/fargate-unzip/Dockerfile/publish_docker.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+#set -nx
+# Execution Instructions
+# must be executed in the same directory of Dockerfile and .sh
+# Get your Account ID
+dockerContainerName="unzip-shell"
+awsRegion="us-east-1"
+docker build -t $dockerContainerName:latest .
+accountId=$(aws sts get-caller-identity --output text | awk '{print $1}')
+aws ecr get-login-password --region $awsRegion | docker login --username AWS --password-stdin $accountId.dkr.ecr.us-east-1.amazonaws.com/
```

Comment:
Same comment as above

Reply:
you should only deploy this once, I can use a similar strategy as the SDLF deploy script of using command lines parameters. What would you think?

Reply:
Sorry what I meant was to make sure the region is overriden in this `$accountId.dkr.ecr.us-east-1.amazonaws.com/`. But if it's modified by the deploy script too then we are ok

Reply:
you're right, I've also modified that.